packages/svelte/src/compiler/phases/3-transform/server/visitors/SvelteHead.js

```diff
@@ -11,6 +11,12 @@ export function SvelteHead(node, context) {
 	const block = /** @type {BlockStatement} */ (context.visit(node.fragment));
 
 	context.state.template.push(
-		b.stmt(b.call('$.head', b.id('$$payload'), b.arrow([b.id('$$payload')], block)))
+		b.stmt(
+			b.call(
+				'$.head',
+				b.id('$$payload'),
+				b.arrow([b.id('$$payload')], block, node.fragment.metadata.is_sync)
+			)
+		)
```

Comment:
we can change this back (not sure where `is_sync` came from in the first place, probably my bad)

```suggestion
		b.stmt(b.call('$.head', b.id('$$payload'), b.arrow([b.id('$$payload')], block)))
```